.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+### PR Checklist
```

Comment:
This is very vague.

Let's use https://github.com/jenkinsci/.github/blob/master/.github/pull_request_template.md